watch-parties/src/main/java/com/google/sps/servlets/Video.java

```diff
@@ -0,0 +1,31 @@
+package com.google.sps.servlets;
+import java.util.ArrayList;
+import java.lang.Integer;
+import java.util.Date;
+import java.io.PrintWriter;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.KeyFactory;
+import com.google.appengine.api.datastore.Key;
+import java.util.PriorityQueue;
+public class Video {
+  private int position;
+  private String videoId;
+  public Video(String url){
+      videoId = url.substring(url.length()-11   );
```

Comment:
We probably want something more robust here. We need to account for links like https://www.youtube.com/watch?v=bD-Benuv_5g&feature=youtu.be&t=2 and links like https://youtu.be/bD-Benuv_5g.

I'm okay with this being handled in a separate PR since this is big. But we'll need a function that can parse the video id from different youtube URL inputs.

Reply:
There might be a URL framework that makes parsing params easier.